csharp/private/providers.bzl

```diff
@@ -102,9 +110,13 @@ FrameworkCompatibility = {
     "netstandard2.1": ["netstandard2.0"],
 
     # .NET Framework
+    "net11": [],
```

Comment:
So the idea here is that we never "fill in" these frameworks with compatible ones (e.g. `net20` -> `net30`) but these can be used to populate the other direction (`net30` -> `net40`). The precedence is the natural order.

Maybe we don't need to do this. I wanted to prevent `@net//:System.Linq` from existing though, because that's funky. It does mean it'd be extra hard to build `net30` things with `csharp_library`, but I don't know that we should really be worried about that.

Reply:
Oh yeah, true, having `net20` provide the standard references for everything else wouldn't be amazing.